gtsam/hybrid/HybridBayesNet.h

```diff
@@ -124,6 +124,37 @@ class GTSAM_EXPORT HybridBayesNet : public BayesNet<HybridConditional> {
   /// Prune the Hybrid Bayes Net such that we have at most maxNrLeaves leaves.
   HybridBayesNet prune(size_t maxNrLeaves);
 
+  /**
+   * @brief 0.5 * sum of squared Mahalanobis distances
+   * for a specific discrete assignment.
+   *
+   * @param continuousValues Continuous values at which to compute the error.
+   * @param discreteValues Discrete assignment for a specific mode sequence.
+   * @return double
+   */
+  double error(const VectorValues &continuousValues,
+               const DiscreteValues &discreteValues) const;
+
+  /**
```

Comment:
Could you (as in the other files, move the tree version *before* the non-tree version? In gtsam, the convention is to call "up", and I prefer the API to be structured that way as well.

Reply:
Semantically, the non-tree version should call the tree version. I don't do that in practice since it is cheaper to index via the discrete values and then compute the error, rather than create the whole tree and index into the tree.